### PR TITLE
remove restriction bountlevel should be < 100

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

**Review by lirona — COMMENTED** [review]
so just that line..

File: test/hatvaults.js [hunk 1]
```diff
@@ -1642,7 +1684,10 @@ contract("HatVaults", (accounts) => {
 
     //stake
     await hatVaults.deposit(0, web3.utils.toWei("1"), { from: staker });
-    assert.equal(await hatToken.balanceOf(hatVaults.address), web3.utils.toWei(hatVaultsExpectedHatsBalance.toString()));
+    assert.equal(
+      await hatToken.balanceOf(hatVaults.address),
```
[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply test/hatvaults.js]: i can merge it in from develop

File: contracts/vaults/Base.sol [hunk 2]
```diff
@@ -354,7 +354,7 @@ contract  Base is Governable, ReentrancyGuard {
 
     /**
     * @dev Check bounty levels.
-    * Each level should be less than `HUNDRED_PERCENT`
+    * Each level should be less than or equal to `HUNDRED_PERCENT`
     * If _bountyLevels length is 0, default bounty levels will be returned ([2000, 4000, 6000, 8000]).
     * @param _bountyLevels The bounty levels array
```
[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply contracts/vaults/Base.sol:359]: oops. And I should have written a test for it

File: test/hatvaults.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (lirona) [reply test/hatvaults.js]: ok, still it was too narrow for me but if you like it that way it's fine

**Review by lirona — COMMENTED** [review]
.

File: test/hatvaults.js [hunk 3]
```diff
@@ -529,19 +529,25 @@ contract("HatVaults", (accounts) => {
     } catch (ex) {
       assertVMException(ex, "HVE19");
     }
+
+    // bountylevel can be 10000 without throwing an error
+    await hatVaults.setPendingBountyLevels(0, [10000], {
```
[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply test/hatvaults.js:534]: if it does not work it throws an error

**Pull request merged**